java/maven4/Portfile

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup select 1.0
+PortGroup java 1.0
+
+name            maven4
+version         4.0.0-rc-1
```

Comment:
I'd recommend using a version of 3.9.9-rc1 instead

Reply:
That would be kind of confusing, because Maven 3.9.9 is the current Maven 3 release. Do you mean that MacPorts' versioning considers 4.0.0-rc-1 newer than 4.0.0? If so, is there no other postfix that would make MacPorts consider it a version before 4.0.0?

Reply:
Using 3.9.9-rc1 isn't a big deal. Plus the ultimate 4.0 release may be out before long.

Reply:
I've changed the port's version to 3.9.9-rc1: https://github.com/macports/macports-ports/pull/27040/commits/240d1c3cc50189100a28f235811ec8c5b7aec7c3